### PR TITLE
Add code owners

### PR DESCRIPTION
**Pull request by wli3**
To have clear team's responsibility.

### CODE REVIEW

**Comment by wli3** [comment]
@dotnet/dotnet-cli @dotnet/nuget-team @dotnet/fsharp @dotnet/dotnet-test-templates-admin @DonJoseLuis @jonsequitur 
(and @nkolev92 could you help me find Arturo's github handle?)

Please verify:
1. If you agree with the ownership
2. If the github handle in the file is the best one.

**Comment by cartermp** [comment]
Totes

**Comment by nkolev92** [comment]
The handle is @aortiz-msft. 

The NuGet team does contain many aliases that are *not* currently on the NuGet team.

**Comment by Forgind** [comment]
> could you help me find Arturo's github handle?)

For future reference: https://repos.opensource.microsoft.com/people (internal link)

File: CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+# @dotnet/dotnet-cli  will be requested for
+# review when someone opens a pull request.
+*       @dotnet/dotnet-cli
+
 /src/WebSdk/ @vijayrkn
-/src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/ @vijayrkn
\ No newline at end of file
+/src/Tests/Microsoft.NET.Sdk.Publish.Tasks.Tests/ @vijayrkn
+
+/src/Cli/dotnet/commands/dotnet-add/dotnet-add-package @dotnet/nuget-team
```
[the remark this answers is not in the document]
Comment (wli3) [reply CODEOWNERS:13]: This is always owned by nuget. We can give you merge permissions.

[the remark this answers is not in the document]
Comment (wli3) [reply CODEOWNERS:13]: Are good on this or we need a meeting to discuss? @aortiz-msft @marcpopMSFT

[the remark this answers is not in the document]
Comment (wli3) [reply CODEOWNERS:13]: Sorry I am confused by your thrum up. We are good? @nkolev92

[the remark this answers is not in the document]
Comment (nkolev92) [reply CODEOWNERS:13]: I was just waiting on @aortiz-msft for a comment :)

**Pull request merged**